.travis.yml

```diff
@@ -24,5 +24,5 @@ script:
     - test -z "$(go fmt .)"
     - go vet .
     - go test -v -race .
-    # Only linux until https://github.com/golang/go/issues/16150 is fixed.
-    - if [[ "$OS" == "linux" ]]; then CGO_CFLAGS="-fsanitize=address" CGO_LDFLAGS="-fsanitize=address -fuse-ld=gold" go test -v .; fi
+    # Disabled until https://github.com/golang/go/issues/16150 is fixed.
+    # CGO_CFLAGS="-fsanitize=address" CGO_CXXFLAGS="-fsanitize=address" CGO_LDFLAGS="-fsanitize=address -fuse-ld=gold" go test -v .
```

Comment:
What happens if you just removed `CGO_CXXFLAGS="-fsanitize=address"` leaving this enabled?

Reply:
- With `CGO_CXXFLAGS="-fsanitize=address"`, it gives the error about `CGO_getFakeDataChannel` (#95).
- Without `CGO_CXXFLAGS="-fsanitize=address"`, it gives the error about `CGO_Channel_Label` (https://github.com/keroserene/go-webrtc/issues/81#issuecomment-443034003, may be the same as #43).

Reply:
The reason I first did a commit adding `CGO_CXXFLAGS="-fsanitize=address"`, before commenting the whole line out, is so that a future `git revert` of the latter commit will have the `CGO_CXXFLAGS` in place, and not require a future maintainer to rediscover that it is supposed to be there.

Reply:
Damned if we do; damned if we don't.